homeassistant/components/icloud/strings.json

```diff
@@ -32,7 +39,8 @@
     },
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
-      "no_device": "None of your devices have \"Find my iPhone\" activated"
+      "no_device": "None of your devices have \"Find my iPhone\" activated",
+      "updated_password": "Your password was successfully updated"
```

Comment:
Use this common string:
https://github.com/home-assistant/core/blob/f268378f0fdf3cccab00d57c13104acfd2447755/homeassistant/strings.json#L68